### ISSUE TITLE
Doesn't work for me

### ISSUE THREAD

**Issue by Oreolek** [tdreyno/middleman-blog-editor-issues#12]
Um, I bought the extension but it doesn't work. How do I start the editor?

**Comment by tdreyno** [comment]
yeah, sorry, this extension is no longer being updated. I'll issue a refund ASAP
